public/js/controllers/data_controller.js

```diff
@@ -147,14 +147,28 @@ export class DataController {
     }
 
     static watchATCredits(projectName, onChange) {
-        let ATCredits = [];
-        return mediadb.collection(`metrics/africas_talking/${projectName}`)
-            .orderBy("datetime", "desc")
-            .limit(1).onSnapshot(res => {
-                // Update data every time it changes in firestore
-                DataController.updateData(res, ATCredits);
-                onChange(ATCredits);
-            }, error => console.log(error));
+        const ref = mediadb.collection("active_projects").doc(projectName);
+        ref.get()
+            .then((doc) => {
+                const data = doc.data();
+                if (!data.hasOwnProperty("africas_talking")) {
+                    onChange("N/A");
+                    return;
+                }
+
+                let ATCredits = [];
+                return mediadb
+                    .collection(`metrics/africas_talking/${projectName}`)
+                    .orderBy("datetime", "desc")
+                    .limit(1)
+                    .onSnapshot((res) => {
+                        // Update data every time it changes in firestore
+                        DataController.updateData(res, ATCredits);
+                        onChange(ATCredits);
```

Comment:
Given that you're requesting up to 1 result, I think it would be slightly cleaner if you passed that result here rather than the list the result lives in

Reply:
Thanks for the suggestion, updated.